projects/RabbitMQ.Client/client/impl/ConsumerWorkService.cs

```
@@ -103,9 +107,9 @@ private async Task Loop()
                         {
                             work();
                         }
-                        catch(Exception)
+                        catch(Exception e)
                         {
-                            // ignored
+                            _channelBase.OnUnhandledExceptionOccurred(e, "ConsumerWorkService");
```

This is the empty catch I patched up